CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.97.0-beta.x
+
+- **Breaking change** Add the passing on an explicit `Registry` to all calls to `createType` as well as all codec classes. If you are upgrading form an older version, use the [migration guide](UPGRADING.md) to walk through the changes required.
+- **Breaking change** The metadata classes have been moved from `@polkadot/types` into `@polkadot/metadata`. If you explicitly create `Metadata` classes, you now would need to do so with the `import { Metadata } from '@polkadot/metadata';`
```

Comment:
We don't export Metadata, should be `import Metadata from '@polkadot/metadata';` (and this one is the decorated now, which should be the user-facing one).

So would also add a note about `Metadata.metadata`